### PR TITLE
[MLU] fluid: add mluop

### PR DESCRIPTION
**Pull request by cifar10**
### PR types
New features

### PR changes
OPs

### Describe
add fluid mluop

### CODE REVIEW

File: paddle/fluid/operators/mlu/mlu_baseop.h [hunk 1]
```diff
@@ -2149,6 +2270,39 @@ class MLUCnnl {
...
+  //     const float step_w,
+  //     const float offset,
+  //     const bool clip,
+  //     const bool min_max_aspect_ratios_order,
+  //     const mluOpTensorDescriptor_t output_desc,
+  //     void *output,
+  //     const mluOpTensorDescriptor_t var_desc,
+  //     void *var);
+
+  static void OpAbs(const ExecutionContext& ctx,
```
Comment (ShawnNew) [inline paddle/fluid/operators/mlu/mlu_baseop.h]: delete

**Review by ShawnNew — DISMISSED** [review]
👍

File: paddle/fluid/operators/mlu/mlu_baseop.cc [hunk 2]
```diff
@@ -5148,5 +5346,54 @@ MLURNNDesc::~MLURNNDesc() {
                                                               diff_x));
 }
 
+// /* static */ void MLUOp::OpPriorBox(
+//     const ExecutionContext& ctx,
```
Comment (ronny1996) [inline paddle/fluid/operators/mlu/mlu_baseop.cc]: 不需要的代码可以删掉

**Review by ronny1996 — CHANGES_REQUESTED** [review]

File: paddle/fluid/operators/mlu/mlu_baseop.cc (hunk 2 above)
Comment (cifar10) [reply paddle/fluid/operators/mlu/mlu_baseop.cc]: 已修改

File: paddle/fluid/operators/mlu/mlu_baseop.h (hunk 1 above)
Comment (cifar10) [reply paddle/fluid/operators/mlu/mlu_baseop.h]: 已修改

**Review by qili93 — APPROVED** [review]
LGTM

**Review by ronny1996 — APPROVED** [review]

**Pull request merged**